Tasks/CondaEnvironmentV1/task.json

```diff
@@ -13,7 +13,7 @@
     "version": {
         "Major": 1,
         "Minor": 140,
```

Comment:
It would be better to change the Minor version to the current sprint number.

Reply:
@jpricketMSFT I'm going to hotfix this to 140 and 141.  We've decided in those cases to set the version according to the earliest sprint it will be backported to

Reply:
@kelliejos